modules/storage/main.tf

```
@@ -32,12 +32,11 @@ resource "google_project_service" "enable_destination_api" {
 # Storage bucket #
 #----------------#
 resource "google_storage_bucket" "bucket" {
-  name               = var.storage_bucket_name
-  project            = google_project_service.enable_destination_api.project
-  storage_class      = var.storage_class
-  location           = var.location
-  force_destroy      = true
-  bucket_policy_only = var.bucket_policy_only
+  name                        = var.storage_bucket_name
+  project                     = google_project_service.enable_destination_api.project
+  storage_class               = var.storage_class
+  location                    = var.location
+  uniform_bucket_level_access = var.bucket_policy_only
```

Since this is a breaking change anyway, is there a reason why we don't want to change this var name?
```suggestion
  uniform_bucket_level_access = var.uniform_bucket_level_access
```

---

Done.